src/RealtimeEffectPanel.cpp

```diff
@@ -310,45 +308,49 @@ namespace
          auto sizer = std::make_unique<wxBoxSizer>(wxHORIZONTAL);
 
          //On/off button
-         auto enableButton = safenew ThemedButtonWrapper<wxBitmapButton>(this, wxID_ANY, wxBitmap{}, wxDefaultPosition, wxDefaultSize, wxNO_BORDER);
-         enableButton->SetBitmapIndex(bmpEffectOn);
+         auto enableButton = safenew ThemedAButtonWrapper<AButton>(this);
```

Comment:
What was the major advantage in this replacement of `wxButton` with `AButton`?

More states that can be associated with changed images?

More convenient control of the button state?

Reply:
Native buttons are not focusable on mac until such behaviour explicitly enabled in the preferences. Native buttons do not support text elision, appearance can't be customized... many reasons actually